packages/pluggableWidgets/badge-web/tests/e2e/specs/dataTypes.spec.ts

```diff
@@ -33,6 +33,13 @@ describe("BadgeButton different data types", () => {
 
             expect(badgeButton.getBadgeText()).toEqual("0.56");
         });
+
+        it("should display correctly enum data", () => {
```

Comment:
should or displays?

Reply:
displays :) I changed it for all test cases in the file.